gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImplCreateTest.java

```diff
@@ -50,7 +50,7 @@ public void create_grpcAndVmComputeEngineCredentials_useDirectpath() throws IOEx
   public void create_grpcAndDisableDirectPathAndVmComputeEngineCredentials_useCloudpath() throws IOException {
     GoogleCloudStorageImpl gcs = new GoogleCloudStorageImpl(GoogleCloudStorageOptions.builder().setAppName("app")
         .setGrpcEnabled(true).setDirectPathPreffered(false).build(), createStorage(),
-        ComputeEngineCredentials.newBuilder().build());
+        ComputeEngineCredentials.newBuilder().build(), null);
```

Comment:
We can not change constructor for public classes in subminor release, we can only add new ones and deprecate old ones.